libs/scss-foundation/tests/borders/border-radius.tests.scss

```diff
@@ -0,0 +1,40 @@
+@use 'sass-true/sass/true';
+@use '../../src/modules/borders/border-radius' as border;
+
+@include true.describe('partial-border-radius($top-left, $top-right, $bottom-right, $bottom-left)') {
+	@include true.it('should set border-radius with default values') {
+		@include true.assert() {
+			@include true.output() {
+				@include border.partial-border-radius();
+			}
+			@include true.expect() {
+				border: {
+					/* stylelint-disable scss/declaration-nested-properties */
+					top-left-radius: 0;
+					top-right-radius: 0;
+					bottom-right-radius: 0;
+					bottom-left-radius: 0;
+					/* stylelint-enable scss/declaration-nested-properties */
+				}
+			}
+		}
+	}
```

Comment:
We could add another test (just for practice sake, this is more than enough for coverage): to test only provision of second and third argument.

Basically, to avoid calling function with many optional parameters, we can try syntax without nulls and with named params such as:
@include border.partial-border-radius($top-right: 10px, $bottom-right: 1.2em);